keyboards/polykybd/lang/_insert_settings_row.py

```
@@ -0,0 +1,158 @@
+    if not settings or settings != list(range(settings[0], settings[-1] + 1)):
+        sys.exit(f"settings rows are not contiguous: {settings}")
+    last, new_row = settings[-1], settings[-1] + 1
+    if any((cell_text(r, 1) or "") == KEY for r in settings):
+        sys.exit(f"{KEY} already exists")
+
+    langs = []                                   # header row 1, one name per 4 columns
+    c = 2
+    while (name := cell_text(1, c)):
+        langs.append(name)
+        c += 4
+    unknown = [l for l in LANGS if l not in langs]
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)